packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -21,13 +21,19 @@ contract SuperchainConfig is Initializable, ISemver {
         GUARDIAN
     }
 
-    /// @notice Whether or not the Superchain is paused.
-    bytes32 public constant PAUSED_SLOT = bytes32(uint256(keccak256("superchainConfig.paused")) - 1);
-
     /// @notice The address of the guardian, which can pause withdrawals from the System.
     ///         It can only be modified by an upgrade.
     bytes32 public constant GUARDIAN_SLOT = bytes32(uint256(keccak256("superchainConfig.guardian")) - 1);
```

Comment:
Honestly I don't see any good reason why we're using unstructured storage here. Maybe we should just transition this whole contract to use structured storage for simplicity. This mix is awkward and confusing.